cristin-import/src/test/resources/features/BookMonographRules.feature

```diff
@@ -38,5 +38,7 @@ Feature:
     Then the NVA Resource has a PublicationContext with publisher equal to "House of Publishing"
 
 
-
-
+  Scenario: Map throws exception when Cristin with type book report has no number of pages value.
```

Comment:
Mapping fails when a Cristin Result of type Monografi has no information about the number of pages